src/join.t.sol

```diff
@@ -380,6 +381,29 @@ contract DssDeployTest is DssDeployTestBase {
         assertEq(vat.gem("UNI", address(this)), 6 ether);
     }
 
+    function testGemJoin_AAVE() public {
+        deployKeepAuth();
+        DSValue pip = new DSValue();
+
+        AAVE aave = new AAVE(100 ether);
```

Comment:
Kind of a nitpicky thing here, but would you mind changing these `ether` usages to `n * 10**18` or `n * WAD`? `ether` is a convenient bit of solidity here but it's kind of a loaded term. It looks like `UNI` is using that pattern too, so we'll need to go back and fix those.